book/figures.js

```diff
@@ -1,6 +1,6 @@
 
```

Comment:
Changes in this file are just because my JavaScript linter must not have been running on it before.